README.md

```diff
@@ -520,6 +520,9 @@ model_config:
     type: # linear | dynamic
     factor: # float
 
+# optional overrides to the bnb quantization configuration
+bnb_config_kwargs:
```

Comment:
Can you provide some examples on what values would look like here?

Reply:
If there is a link to the bnb options could be good for a comment too, in addition to examples?

Reply:
Sure!

Reply:
updated! Thank you!